tools/lint/rules.py

```diff
@@ -86,6 +86,11 @@ class AhemCopy(Rule):
     description = "Don't add extra copies of Ahem, use /fonts/Ahem.ttf"
 
 
+class LicenseFiles(Rule):
+    name = "LICENSE"
+    description = "Don't add new license files"
```

Comment:
I think we need more context here. Maybe include "Do not add new third-party code without going through the RFC process". Otherwise, people might just add dependencies without `LICENSE`.